src/app/plugins/quote.plugin.ts

```diff
@@ -56,11 +56,7 @@ export default class QuotePlugin extends Plugin {
       .setTitle(tag === 'Famous-quotes' ? 'Famous-Quotes' : tag)
       .setFooter('Pulled from api.quotable.io')
       .setTimestamp(new Date())
-      .setDescription(
-        `${content}
-
-        - ${author}`
-      );
+      .setDescription(content + '\n\n- ' + author);

```

Comment:
String interpolation should be used here.

```suggestion
      .setDescription(`${content}\n\n-${author}`);
```

Reply:
is there a specific reason to use it over a regular string?

Reply:
It's more concise, and allows easier modification of the string, if needed, down the line.